### DISCUSSION TITLE
Pytest 7.0.0 appears to break pytest-pythonpath python_paths in pytest.ini

### DISCUSSION THREAD

**Discussion by enewton** [pytest-dev/pytest#9635] [Q&A]
We have some utility helper code in a separate folder to our tests. Up until now we have this configured via pytest.ini like so:

```
[pytest]
python_paths = ./utils/
```

This definition appears to be getting ignored after upgrading to pytest 7.0.0, which then gives us 'module not found' errors on tests trying to import utils. Reading around, it looks like this functionality is now built in to pytest and so is pytest-pythonpath obsolete?

I can get our tests passing again by updating to:

```
[pytest]
pythonpath = ./utils/
```

So the main question is, is this change in behaviour expected? Has the old method been deliberately disabled or should I raise a bug?

**Comment by bluetech** [comment]
> Reading around, it looks like this functionality is now built in to pytest and so is pytest-pythonpath obsolete?

Yes, pytest-pythonpath was integrated into pytest itself more or less.

> is this change in behaviour expected?

No, I don't remember anything that should break pytest-pythonpath. I will create an issue from this discussion to investigate this.

**Reply by enewton** [reply]
Thanks for looking into this. As a clue, when running 7.0.0 I get a warning:
`PytestConfigWarning: Unknown config option: python_paths`
Which suggests the pythonpath plugin hasn't loaded?

I've created a simple reproducer. Out of the box, pytest==6.2.5 runs for me. With pytest==7.0.0 I have to uncomment the line in pytest.ini
[pytest_repro.zip](https://github.com/pytest-dev/pytest/files/8015261/pytest_repro.zip)

**Reply by bluetech** [reply]
Thanks, I was able to figure out the problem, see #9636.

**Reply by enewton** [reply]
Thank you. That makes sense. And so is the change of key from `python_paths` to `pythonpath` an intentional attempt to avoid conflicts? Are they semantically subtly different or could the built-in plugin just have read `python_paths`?

**Reply by bluetech** [reply]
It was intended to avoid conflicting with pytest-pythonpath, and also to match the way Python spells it - `PYTHONPATH`.
